src/include/daos_srv/control.h

```diff
@@ -14,21 +14,77 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include <assert.h>
+#include <string.h>
 
-/*
+/**
  * Space separated string of CLI options to pass to DPDK when started during
  * spdk_env_init(). These options will override the DPDK defaults.
  */
 extern const char *
 dpdk_cli_override_opts;
 
-/* Device state flags */
+/** Device state flags */
 #define NVME_DEV_FL_PLUGGED	(1 << 0)
 #define NVME_DEV_FL_INUSE	(1 << 1) /* Used by DAOS (present in SMD) */
 #define NVME_DEV_FL_FAULTY	(1 << 2)
 #define NVME_DEV_FL_IDENTIFY	(1 << 3) /* SSD being identified by LED activity */
 
-/*
+/** Device state combinations */
+#define NVME_DEV_STATE_NORMAL (NVME_DEV_FL_PLUGGED | NVME_DEV_FL_INUSE)
+#define NVME_DEV_STATE_FAULTY (NVME_DEV_STATE_NORMAL | NVME_DEV_FL_FAULTY)
+#define NVME_DEV_STATE_NEW (NVME_DEV_FL_PLUGGED)
+
+#define BIT_SET(x,m) (((x)&(m)) == (m))
+#define BIT_UNSET(x,m) (!BIT_SET(x,m))
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)